src/ML-SNAP/compute_snap.cpp

```diff
@@ -13,7 +13,6 @@
 ------------------------------------------------------------------------- */
 
 #include "compute_snap.h"
-
```

Comment:
Please don't delete this line. It is part of the include file conventions and required once we enable clang-format globally, so that it doesn't sort the class header away from the top spot.